### PR TITLE
TRA-4256 gui oas window

### PR DESCRIPTION
**Pull request by AmitUp9**

### CODE REVIEW

File: ui/src/components/UI/Tabs.tsx [hunk 1]
```diff
@@ -40,7 +39,7 @@ const useTabsStyles = makeStyles((theme) => ({
     },
 
     active: {
-        fontWeight: theme.typography.fontWeightBold,
+        fontWeight: 600,
```
Comment (leon-up9) [inline ui/src/components/UI/Tabs.tsx:42]: isthat intentional checkin?
is it working?

File: ui/src/components/OasModal/OasModal.tsx [hunk 2]
```diff
@@ -1,44 +1,121 @@
...
+  bgcolor: 'background.paper',
+  borderRadius: '5px',
+  boxShadow: 24,
+  p: 4,
+  color: '#000',
+};
 
 const api = Api.getInstance();
-const noOasServiceSelectedMessage = "Please Select OasService";
+const ipAddressWithPortRegex = new RegExp('([0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}):([0-9]{1,5})');
```
Comment (leon-up9) [inline ui/src/components/OasModal/OasModal.tsx:26]: import from utils.ts

File: ui/src/components/OasModal/OasModal.tsx [hunk 3]
```diff
@@ -1,44 +1,121 @@
...
+
+const redocThemeOptions = {
```
Comment (leon-up9) [inline ui/src/components/OasModal/OasModal.tsx:28]: is it can be imported?

Comment (AmitUp9) [reply ui/src/components/OasModal/OasModal.tsx:28]: Where is the best place to put it?

File: ui/src/components/OasModal/OasModal.tsx [hunk 4]
```diff
@@ -1,44 +1,121 @@
...
+import { Box, Fade, FormControl, MenuItem, Modal, Backdrop, ListSubheader } from "@material-ui/core";
+import { useCallback, useEffect, useState } from "react";
 import { RedocStandalone } from "redoc";
 import Api from "../../helpers/api";
 import { Select } from "../UI/Select";
 import closeIcon from "../assets/closeIcon.svg";
 import { toast } from 'react-toastify';
-import './OasModal.sass'
+import style from './OasModal.module.sass';
+import opnApiLogo from '../assets/openApiLogo.png'
```
Comment (leon-up9) [inline ui/src/components/OasModal/OasModal.tsx:9]: openApiLogo - typo

File: ui/src/components/UI/Tabs.tsx (hunk 1 above)
Comment (AmitUp9) [reply ui/src/components/UI/Tabs.tsx:42]: its working. theme doesn't seems to work

**Review by leon-up9 — APPROVED** [review]

**Pull request merged**